node/tests/GlideClusterClient.test.ts

```diff
@@ -1955,6 +1955,68 @@ describe("GlideClusterClient", () => {
         TIMEOUT,
     );
 
+    it.each([ProtocolVersion.RESP2, ProtocolVersion.RESP3])(
+        "should handle connection timeout when client is blocked by long-running command (protocol: %p)",
+        async (protocol) => {
```

Comment:
you can probably do this test in shared tests to avoid code duplicating

Reply:
afaik we split all tests requiring to create a client